exporters/jaeger/agent.go

```diff
@@ -126,6 +126,11 @@ func (a *agentClientUDP) EmitBatch(ctx context.Context, batch *gen.Batch) error
 		// drop the batch if serialization of process fails.
 		return err
 	}
+
+	maxPacketSize := a.maxPacketSize
+	if maxPacketSize == udpPacketMaxLength {
+		maxPacketSize -= emitBatchOverhead
```

Comment:
```suggestion
	if maxPacketSize > udpPacketMaxLength - emitBatchOverhead {
		maxPacketSize = udpPacketMaxLength - emitBatchOverhead
```

If `maxPacketSize` were `udpPacketMaxLength - 1` there could be problems.

Reply:
I thought the same, but it also appears to currently be valid to set `maxPacketSize > udpPacketMaxLength` in your config.  If we also validate that `maxPacketSize <= udpPacketMaxLength`, then I think this makes sense.

Reply:
Yes, I agree with that. Adjusted!